config/application.rb

```diff
@@ -24,6 +24,10 @@ class Application < Rails::Application
     config.assets.paths <<
       Rails.root.join("vendor", "assets", "bower_components")
 
+    config.angular_templates.module_name    = 'templates'
```

Comment:
Unnecessary spacing detected.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.